pkg/api/deploymentapi/get.go

```diff
@@ -68,17 +68,31 @@ func Get(params GetParams) (*models.DeploymentGetResponse, error) {
 		return nil, err
 	}
 
+	requestParams := deployments.NewGetDeploymentParams().
+		WithDeploymentID(params.DeploymentID).
+		WithShowPlans(ec.Bool(params.ShowPlans)).
+		WithShowPlanDefaults(ec.Bool(params.ShowPlanDefaults)).
+		WithShowPlanLogs(ec.Bool(params.ShowPlanLogs)).
+		WithShowPlanHistory(ec.Bool(params.ShowPlanHistory)).
+		WithShowMetadata(ec.Bool(params.ShowMetadata)).
+		WithShowSettings(ec.Bool(params.ShowSettings)).
+		WithConvertLegacyPlans(ec.Bool(params.ConvertLegacyPlans)).
+		WithShowSystemAlerts(systemAlerts)
+	if params.ShowInstanceConfigurations {
```

Comment:
Why can't we use the same style as for, e.g. `ShowPlans`? E.g.
```
.WithShowInstanceConfigurations(ec.Bool(params.ShowInstanceConfigurations))
```
instead of the explicit `if`?

Reply:
The difference is that the if only adds the parameter to the request if it is set to true (assuming the default in the API is false as well). The other parameters will be added to every request.

My reasoning behind it will only add the parameter to the request when it is needed.

Another option would be to make the parameters `*bool` and pass them straight in. Then it is still possible in the `cloud-sdk-go` api to decide to either set the parameter, or not set it and it will not be added.